architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,43 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with {product-title} 3.11, you can specify Security Context Constrains as a
+resource that is handled by RBAC. This allows you to
+scope the access to your SCCs to a certain project or cluster wide. Assigning users/groups
+directly to an SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, you need to specify the following rule:
+.Role-Based Access to SCCs
+[source,yaml]
+----
+rules:
+  apiGroups:
+  - security.openshift.io <1>
+  resources:
+  - securitycontextconstraints <2>
+  verbs:
+  - use
+  resourceNames:
+  - myPermittingSCC <3>
+----
+<1> The API group which includes the `securitycontextconstraints` resource
+<2> Name of the resource group which allows us to specify SCC names in the
+`resourceNames` field
+<3> An example name for an SCC we want to give access to
+
+A role, or cluster role, with such a rule would allow the subjects that are bound to it with a
+rolebinding or a clusterrolebinding to use the user-defined SCC called
+*myPermittingSCC*.
+
+[NOTE]
+====
+RBAC is deisgned to prevent escalation, even project administrators will be
```

Comment:
I'd say:

Because RBAC is designed to prevent escalation, you must manually grant access to SCCs. Not even
project administrators can grant access to an SCC. By default, they lack permission to add the verb *use* to any SCC, including the *restricted* SCC.

Reply:
I think the original description is a bit more accurate. Adding "Because" before it, though, it feels missing there.